packages/core-generator/src/types.ts

```diff
@@ -29,6 +29,7 @@ export interface toStringOptions {
   componentInputs?: ComponentInput[];
   isComponent?: boolean;
   fromType?: boolean;
+  mutableOptions?: { hasRestAttributes?: boolean };
```

Comment:
To my mind mutableOptions should be placed in the specific type of the angular-generator.

Reply:
[See previous comment](https://github.com/DevExpress/devextreme-renovation/pull/870#discussion_r804412717)